tests/scripts/run_performance.sh

```diff
@@ -88,6 +92,8 @@ run_device_perf_models() {
 
```

Comment:
It looks like some the tests on this list are not managing device switch pretty well. 

For example bert is being attempted here: 
https://github.com/tenstorrent-metal/tt-metal/actions/runs/8622196945/job/23632807803#step:7:1367
and it is crashing. Even if it wasn't crashing its expected numbers are for GS and not WHB0. 

I think we should check for device arch here as well, similar to what you have done on line 75 for models.

Reply:
Or we can tag all the test here properly. This might actually be the cleaner option.

Reply:
Sounds good yeah thanks